Dockerfile-operator

```diff
@@ -13,10 +13,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM confluentinc/cp-kafka-connect:5.3.1
+ARG CP_VERSION
 
```

Comment:
Do we know why `ENV CONNECT_PLUGIN_PATH` is not required for `confluentinc/cp-server-connect-operator` but is required for the other case?

Reply:
I do not